setup.py

```diff
@@ -35,9 +35,9 @@ def read(*parts):
 
 
 install_requires = [
-    'django>=1.8,<2.0',
+    'django>=1.11,<3',
```

Comment:
Remove support for django<1.11 as those versions are too old and not used much. Projects using django<1.11 can add version constraint until they have upgraded their django version.